books/free-programming-books-el.md

```diff
@@ -7,7 +7,7 @@
 * [Python](#python)
 * [Scala](#scala)
 * [SQL](#sql)
-
+*.[kotlin](#kotlin)
```

Comment:
this should be `* [Kotlin](#kotlin)` and write after `[JavaScript] (after line 6)`